lib/ansible/modules/cloud/docker/docker_swarm_service.py

```diff
@@ -214,7 +214,7 @@
             C(T) (tebibyte), or C(P) (pebibyte)."
           - C(0) equals no reservation.
```

Comment:
Good catch! 

This should also be changed to describe limits instead of reservations.
```suggestion
          - C(0) equals no limit.
```

Reply:
Oh, true! Commit amended with this change. Thanks!